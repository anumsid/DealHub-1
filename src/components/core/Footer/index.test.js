import React from 'react'
import { shallow } from 'enzyme'
import { stub } from 'sinon'
import toJson from 'enzyme-to-json'
import { Footer } from './index'

describe('Footer Component', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Footer />)
  })

  it('Matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
