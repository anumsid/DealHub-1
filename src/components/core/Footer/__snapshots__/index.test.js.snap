// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Matches snapshot 1`] = `
<footer
  className="page-footer"
>
  <nav
    className="navbar fixed-bottom"
  >
    <p
      className="navbar-brand"
    >
       Â© 2018 DealHub
    </p>
  </nav>
</footer>
`;
